reinforcement_learning/rlclientlib/live_model_impl.h

```diff
@@ -76,6 +79,7 @@ namespace reinforcement_learning
     std::unique_ptr<i_logger> _ranking_logger{nullptr};
     std::unique_ptr<i_logger> _outcome_logger{nullptr};
     std::unique_ptr<model_management::model_downloader> _model_download{nullptr};
+    std::unique_ptr<i_trace> _trace_logger { nullptr };
```

Comment:
>r { nullptr }; [](start = 41, length = 14)

remove spaces to be consistent with above

Reply:
done.